File: 1_Java_Examples/1_ConferenceAttendees_SRP/Attendee.java
public class Attendee {
	
	private String name;
	private AttendeeType type;

	public Attendee(String name, AttendeeType type) {
		this.name = name;
		this.type = type;
	}
	
	public String getName() {
		return this.name;
	}
	
	public AttendeeType getType() {
		return this.type;
	}
	
}

File: 4_Clojure_Examples/1_ConferenceAttendees_SRP/attendee.clj
(ns attendee)

(defn make-attendee [name type]
	{:name name, :type type})

(defn make-participant [name]
	(make-attendee name :participant))

(defn make-speaker [name]
	(make-attendee name :speaker))

(defn make-organizer [name]
	(make-attendee name :organizer))
	
